definitions/web_app.rb

```diff
@@ -37,7 +37,7 @@
       :params           => params
     )
     if ::File.exist?("#{node['apache']['dir']}/sites-enabled/#{application_name}.conf")
-      notifies :reload, 'service[apache2]', :delayed
+      notifies :reload, 'service[apache2]', params[:reload_speed]
```

Comment:
The web_app is not enabled until the apache_site definition below kicks in. As such, I think this notify would be better placed on apache_site below. However because the apache_site is a definition notifications will not work.